cloud/linode/loadbalancers.go

```diff
@@ -601,12 +601,26 @@ func (l *loadbalancers) buildLoadBalancerRequest(ctx context.Context, clusterNam
 	return l.createNodeBalancer(ctx, clusterName, service, configs)
 }
 
+func coerceString(s string, minLen, maxLen int, padding string) string {
+	if len(padding) == 0 {
+		padding = "x"
+	}
+	if len(s) > maxLen {
+		return s[:maxLen]
+	} else if len(s) < minLen {
+		return coerceString(fmt.Sprintf("%s%s", padding, s), minLen, maxLen, padding)
```

Comment:
do we have to recurse here? can't we set the padding to minLen-len(s) ?

Reply:
It's a safeguard against small or large padding arguments. Rather than change this function to also return an error and have to handle that, I chose to recurse after adding the padding so that: a) more padding can be added if needed, or b) the string can be trimmed if the padding made it too long somehow.